docs/en/administration/sync.md

```diff
@@ -1,4 +1,243 @@
+juicefs sync --exclude '.*' /mnt/jfs/ s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com/
+```
+
+Repeat this option with different parameters to exclude multiple patterns. For example, exclude all hidden files, `pic/` directory and `4.png` file:
+
+```shell
+juicefs sync --exclude '.*' --exclude 'pic/' --exclude '4.png' /mnt/jfs/ s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com
+```
+
+#### Include Directories/Files
+
+Option `--include` is for including patterns you don't want to exclude. For example, synchronize only `pic/` and `4.png` and exclude all the others:
+
+```shell
+juicefs sync --include 'pic/' --include '4.png' --exclude '*' /mnt/jfs/ s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com
+```
+
+:::info NOTICE
+The earlier options have higher priorities than the latter ones. Your `--include` options should appear before `--exclude`. All `--include` options such as `--include 'pic/' --include '4.png'` which appear later than `--exclude '*'` will be ignored.
+:::
+
+### Multi-threading and Bandwidth Throttling
+
+The subcommand `sync` enables 10 threads by default. You can customize thread count by `--thread` option.
+
+In addition, you can set option `--bwlimit` in the unit `Mbps` to limit the bandwidth used by the synchronization. Its default value `0` means no limitation.
+
+### Directory Structure and File Permissions
+
+The subcommand `sync` only synchronizes file objects and directories containing file objects, and skips empty directories by default. To synchronize empty directories, you can use `--dirs` option.
+
+In addition, when synchronizing between file systems such as local, sftp and hdfs, option `--perms` can be used to synchronize file permissions from the source to the destination.
+
+### Copy Symbolic Links
+
+You can use `--links` option to disable symbolic link resolving when synchronizing **local directories**. That is, synchronizing only the symbolic links themselves rather than the directories or files they are pointing to. The new symbolic links created by the synchronization refer to the same paths as the original symbolic links without any conversions, no matter whether their references are reachable before or after the synchronization.
+
+Some details need to be noticed
+
+1. The `mtime` of a symbolic link will not be synchronized;
+2. `--check-new` and `--perms` will be ignored on synchronizing symbolic links.
+
+### Multi-machine Concurrent Synchronization
+
+Synchronizing between two object storages is essentially pulling data from one and pushing it to the other. As shown in the figure below, the efficiency of synchronization depends on the bandwidth between the client and the cloud.
+
+![](../images/juicefs-sync-single.png)
+
+When synchronizing a huge amount of data, the synchronization is often bottlenecked since the client machine runs out of bandwidth. For which case JuiceFS Sync provides a multi-machine concurrent solution, as shown in the figure below.
```

Comment:
```suggestion
When synchronizing a huge amount of data, there is often a bottleneck in the synchronization since the client machine runs out of bandwidth. For this case, JuiceFS Sync provides a multi-machine concurrent solution, as shown in the figure below.
```